pkg/services/secrets/kvstore/kvstore.go

```diff
@@ -40,7 +40,7 @@ func ProvideService(
 		secretsPlugin, err = StartAndReturnPlugin(pluginsManager, ctx)
 		namespacedKVStore := GetNamespacedKVStore(kvstore)
 		if err != nil || secretsPlugin == nil {
-			logger.Error("failed to start remote secrets management plugin", "msg", err.Error())
```

Comment:
This was removed because the "msg" returned was "unrecognized message from plugin" it's better to delagate error logging to the plugin which can properly detail the error.

Reply:
gotcha, I missed that this was done in conjunction with the plugin changes, sounds good.